\doxysection{Référence de la classe pip.\+\_\+internal.\+exceptions.\+Configuration\+Error}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1ConfigurationError}{}\label{classpip_1_1__internal_1_1exceptions_1_1ConfigurationError}\index{pip.\_internal.exceptions.ConfigurationError@{pip.\_internal.exceptions.ConfigurationError}}


Graphe d\textquotesingle{}héritage de pip.\+\_\+internal.\+exceptions.\+Configuration\+Error\+:
% FIG 0


Graphe de collaboration de pip.\+\_\+internal.\+exceptions.\+Configuration\+Error\+:
% FIG 1


\doxysubsection{Description détaillée}
\begin{DoxyVerb}General exception in configuration\end{DoxyVerb}
 

La documentation de cette classe a été générée à partir du fichier suivant \+:\begin{DoxyCompactItemize}
\item 
env/lib/python3.\+12/site-\/packages/pip/\+\_\+internal/\mbox{\hyperlink{pip_2__internal_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
